tests/test_decorator.py

```diff
@@ -9,6 +11,7 @@
 
 OPTIONS = [
     ('admin_order_field', 'radius'),
+    ('admin_order_field', Concat('first_name', Value(' '), 'last_name')),
```

Comment:
👍  Tricky bit is, that we test mypy using some python code which is encoded as a string:

- https://github.com/escaped/django-admin-display/blob/52dbef44dc5804c45e1d748088e32568fddb9c90/tests/test_mypy.py#L10
- https://github.com/escaped/django-admin-display/blob/52dbef44dc5804c45e1d748088e32568fddb9c90/tests/test_mypy.py#L29

As we use `OPTIONS` in the corresponding parametrize, we need to add the imports for `Value` and `Concat` there as well. This should fix the error stated in the logs

![image](https://user-images.githubusercontent.com/318650/97603020-3dbf4c00-19c9-11eb-9161-18a3d7905111.png)